xmppserver/src/test/java/org/jivesoftware/Fixtures.java

```diff
@@ -217,4 +250,58 @@ public static IQ iqFrom(final String stanza) {
         }
     }
 
+    /**
+     * Self-Signed invalid X509 certificate for testing purposes (expired 17th June 2023)
+     */
+    public static final String invalidX509Certificate
+        = "-----BEGIN CERTIFICATE-----\n" +
+        "MIICsjCCAZoCCQDsFzeWUN/PbjANBgkqhkiG9w0BAQsFADAbMRkwFwYDVQQDDBB0\n" +
+        "ZXN0LnhtcHAuZG9tYWluMB4XDTIzMDYxNjEzMzE0OFoXDTIzMDYxNzEzMzE0OFow\n" +
+        "GzEZMBcGA1UEAwwQdGVzdC54bXBwLmRvbWFpbjCCASIwDQYJKoZIhvcNAQEBBQAD\n" +
+        "ggEPADCCAQoCggEBAKxaCt4vSjqzXwCfui+S0KjnQrxVagDKJOHbyhkxTbRROJYz\n" +
+        "7SmdAUVHCcFlugOk7UhxTBZ7hHeC3DQTvqilwISRsgyOM8MSAXKV6lvWu2WDRI7s\n" +
+        "LRg5o6r23Me/kiSMXGpzaWitxMOgZWxYJlLb7CfIJwN16F6UvKsX6npN5ETnvzkV\n" +
+        "PZNgoWvy/TX2QlPJTiukX0a4FbyX6REkAgtI6WfLJm7lqtJZFw7KoX2g8GO+wk3v\n" +
+        "akeuuA8OIrtRg5eP5K82a//sF1VoCh9vOryr4mT2BTa8L6x+bF27WMc8+QzXf0JL\n" +
+        "s+Iy8J5dneQWEZPK13Eh5doE59EBx33fadT90CsCAwEAATANBgkqhkiG9w0BAQsF\n" +
+        "AAOCAQEAQn05d+0QjKH5osqz7ZKm7wle/pF1KkOmCD9lfU1+Iu02Adz2nUx+WTaH\n" +
+        "dtB8MtkLQFMcqz6oYquD0ruE3KUvj/A5fmir8wz0m9MG/i3QNrytWepv4vlrcmMr\n" +
+        "yfZLwSrR3UNcNYk9W01/xjQVgH2JsF1B1nbn7eJt0mzr0arHp7VjtjdDJIfkZEEh\n" +
+        "5ZpOERffIINoEptoKMCfjbcp2+PLZ1TjL6MxtrVs+TQmX4i9wL03uNgItbFqYeYP\n" +
+        "RVeb9OaNj4NRMZB49D/jIaqmQWZi6u2ooOQv6AlyzeKInWm+aDmxCAX4IZAod18/\n" +
+        "1hI2qIN8Xj9GUT7wldL368Dq1fUI5g==\n" +
+        "-----END CERTIFICATE-----\n";
+
+    /**
+     * Private key for {@link #invalidX509Certificate} for testing purposes
+     */
+    public static final String invalidX509CertificatePrivateKey
```

Comment:
Maybe rename to `privateKeyForExpiredCert` to reduce the likelyhood that someone (like me) assumes that this private key is somehow invalid (which it isn't, judging by the javadoc).

Reply:
done